src/interfaces/assistants_web/src/hooks/swipeable.ts

```diff
@@ -0,0 +1,23 @@
+import { useEffect } from 'react';
+import { useSwipeable as useSwipe } from 'react-swipeable';
+
+import { useSettingsStore } from '@/stores';
+
+export const useSwipeable = () => {
+  const { setAgentsLeftSidePanelOpen, setAgentsRightSidePanelOpen } = useSettingsStore();
+  const { ref } = useSwipe({
+    onSwipedLeft: ({}) => {
+      setAgentsLeftSidePanelOpen(false);
```

Comment:
Nitpick but can these be organized the same in both blocks. It's easier to read with low context. 

```
 onSwipedLeft: ({}) => {
      setAgentsRightSidePanelOpen(true);
      setAgentsLeftSidePanelOpen(false);
    },
    onSwipedRight: ({}) => {
      setAgentsRightSidePanelOpen(false);
      setAgentsLeftSidePanelOpen(true);
    },
```